01-wasmcloud_basics/wash_intro/claims.md

```diff
@@ -0,0 +1,47 @@
+## wash claims
+
+The `wash claims` subcommand is used for generating and managing [JSON Web Tokens](https://jwt.io) (JWT) for wasmcloud actors.
+
+### wash claims sign
+
+The `wash claims sign` command allows the user to sign a wasm file and couple it with wasmCloud capabilities.
+
+> Note: We have included an unsigned `wasm` module within the Katacoda environment named `table_tennis.wasm`. In a different tutorial, we will edit this file, resign it, and deploy the changes without downtime!
+
+#### Example
+
+`claims sign ./table_tennis.wasm -c wasmcloud:httpserver --name "demo" --ver 0.1.0 --rev 0`{{execute}}
```

Comment:
Ignore if this is what Kevin already commented on , but we can replace `-c wasmcloud:httpserver` with `--http_server`